src/content/self-hosted/helm-configuration.mdx

```diff
@@ -189,6 +189,23 @@ autoscaler:
   slackWebhook:
 ```
 
+### backend
+
+The backend service in Okteto handles the core functionalities of your application.
+
+- `image`: Specifies the image settings for your backend service.
+  - `registry`: The registry where the backend image is hosted. Defaults to `` (You can change all the registries by setting `globals.registry` to your registry).
```

Comment:
Where should i put `(You can change all the registries by setting `globals.registry` to your registry)` message?